src/client/sandbox/native-methods.js

```diff
@@ -195,6 +195,13 @@ class NativeMethods {
 
         this.messageEventOriginGetter = win.Object.getOwnPropertyDescriptor(window.MessageEvent.prototype, 'origin').get;
 
+        if (win.PerformanceEntry) {
+            const namePropDescripor = win.Object.getOwnPropertyDescriptor(window.PerformanceEntry.prototype, 'name');
+
+            if (namePropDescripor && namePropDescripor.get && namePropDescripor.configurable)
```

Comment:
Try to simplify as:
`this.performanceEntryNameGetter = win.Object.getOwnPropertyDescriptor(window.PerformanceEntry.prototype, 'name').get`